crits/core/views.py

```diff
@@ -1390,9 +1390,19 @@ def user_context(request):
     """
 
     context = {}
-    # Get user theme
-    user = CRITsUser.objects(username=request.user.username).first()
-    if user:
+    user = request.user
+    if not user.username:
+        # Get user theme
+        user = CRITsUser.objects(username=request.user.username).first()
```

Comment:
How can we use `request.user.username` when we just checked if we don't have `user.username` which is the same thing?